lib/index.ts

```diff
@@ -16,9 +16,10 @@ declare const BUILD_VERSION: string;
 /** Configuration arguments which include the Descope core SDK args and an optional management key */
 type NodeSdkArgs = Parameters<typeof createSdk>[0] & {
   managementKey?: string;
+  publicKey?: string;
 };
 
-const nodeSdk = ({ managementKey, ...config }: NodeSdkArgs) => {
+const nodeSdk = ({ managementKey, publicKey, ...config }: NodeSdkArgs) => {
```

Comment:
Can this break compatibility?

Reply:
no, if you don't provide - it will not do anything